packages/core/src/alert/alert.element.ts

```diff
@@ -18,16 +18,13 @@ import {
   internalProperty,
   id,
 } from '@clr/core/internal';
-import {
-  CdsIcon,
-  ClarityIcons,
-  timesIcon,
-  infoStandardIcon,
-  successStandardIcon,
-  warningStandardIcon,
-  errorStandardIcon,
-  helpIcon,
-} from '@clr/core/icon';
+import { CdsIcon } from '@clr/core/icon/icon.element.js';
+import { ClarityIcons } from '@clr/core/icon/icon.service.js';
+import { infoStandardIcon } from '@clr/core/icon/shapes/info-standard.js';
+import { successStandardIcon } from '@clr/core/icon/shapes/success-standard.js';
+import { warningStandardIcon } from '@clr/core/icon/shapes/warning-standard.js';
+import { errorStandardIcon } from '@clr/core/icon/shapes/error-standard.js';
+import { helpIcon } from '@clr/core/icon/shapes/help.js';
```

Comment:
This is an internal optimization for ESM tree-shaking. Instead of resolving to `@clr/core/icon` we pull in the icon path directly. This allows the browser to skip the `index.js` which will cause all the icons to be loaded/resolved when using native esm. Most users should continue to use the standard top level import.

```typescript
import { ClarityIcons, userIcon } from `@clr/core/icon`;
```